docs/notebooks/Migration_Guide_beginner.ipynb

```diff
@@ -0,0 +1,448 @@
+          "name": "stderr"
+        },
+        {
+          "output_type": "stream",
+          "text": [
+            "\n",
+            "7-Zip (a) [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21\n",
+            "p7zip Version 16.02 (locale=en_US.UTF-8,Utf16=on,HugeFiles=on,64 bits,2 CPUs Intel(R) Xeon(R) CPU @ 2.20GHz (406F0),ASM,AES-NI)\n",
+            "\n",
+            "\n",
+            "Extracting archive: \n",
+            "--\n",
+            "Path = \n",
+            "Type = tar\n",
+            "Code Page = UTF-8\n",
+            "\n",
+            "Everything is Ok\n",
+            "\n",
+            "Folders: 23\n",
+            "Files: 13397\n",
+            "Size:       107794109\n",
+            "Compressed: 6872064\n"
+          ],
+          "name": "stdout"
+        },
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "mapobs((input = FastAI.Datasets.loadfile, target = FastAI.Datasets.var\"#27#32\"()), DataSubset(::FileDataset, ::Vector{Int64}, ObsDim.Undefined())\n",
+              " 13394 observations)"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 9
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "OGOkrUK1y3D-"
+      },
+      "source": [
+        "```datasetpath()``` function downloads the given dataset and returns the path of the dataset. It consists of all FastAI datasets from python. Each dataset is referred by an unique keyword. The keywords for common datasets are,\n",
+        "\n",
+        "|                      Dataset                      |      Keyword      |\n",
+        "|:-------------------------------------------------:|:-----------------:|\n",
+        "|                       MNIST                       |    \"mnist-png\"    |\n",
+        "|                      CIFAR10                      |     \"cifar10\"     |\n",
+        "|                      Food 101                     |     \"food-101\"    |\n",
+        "|                  Oxford-IIIT Pets                 | \"oxford-iiit-pet\" |\n",
+        "|                     Imagenette                    |    \"imagenette\"   |\n",
+        "|                     Imagewang                     |    \"imagewang\"    |\n",
+        "|                     Imagewoof                     |    \"imagewoof\"    |\n",
+        "|            IMDB Large Movie review data           |       \"imdb\"      |\n",
+        "|                    Wikitext-103                   |   \"wikitext-103\"  |\n",
+        "| CamVid: Motion based Segmentation and Recognition |      \"camvid\"     |\n",
+        "|           PASCAL: Visual Object Classes           |    \"pascal-voc\"   |"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "EmDKNgh2A3Dp"
+      },
+      "source": [
+        "# DataLoaders"
+      ]
+    },
```

Comment:
My concern with this section is that it blurs the line between data containers and data iterators. For a migration guide, I would prefer to see an explanation of how in FastAI.jl, we have a single interface — `MLDataPattern.getobs` and `MLDataPattern.nobs` — for both data containers and data iterators. The only distinction is that data iterators like `eachbatch` or  `eachobs` also implement the [iteration interface](https://docs.julialang.org/en/v1/manual/interfaces/#man-interface-iteration) from Base.

This should be contrasted with fast.ai where there is a distinction between datasets and data loaders.